### PR TITLE
6338 use csrfManager instead of csrfProvider

### PR DESCRIPTION
**Pull request by snoek09**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7 |
| Fixed tickets | #6338 |

### CODE REVIEW

**Comment by WouterJ** [comment]
I don't think we should include the complete setup. Instead use `// ...` to indicate something is missing (maybe add `// ... (see the previous CSRF Protection section for more information)`).

File: components/form/introduction.rst [hunk 1]
```diff
@@ -202,9 +206,17 @@ to bootstrap or access Twig and add the :class:`Symfony\\Bridge\\Twig\\Extension
     )));
     $formEngine = new TwigRendererEngine(array($defaultFormTheme));
     $formEngine->setEnvironment($twig);
+
+    // create a Session object from the HttpFoundation component
+    $session = new Session();
+
+    $csrfGenerator = new UriSafeTokenGenerator();
+    $csrfStorage = new SessionTokenStorage($session);
```
Comment (xabbuh) [inline components/form/introduction.rst]: I wonder if it wasn't better to use the `NativeSessionTokenStorage` to not depend on the HttpFoundation component here.

**Comment by WouterJ** [comment]
:+1: Thanks!

**Comment by xabbuh** [comment]
Thank you @snoek09.

**Pull request closed without merging**